vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -55,13 +55,38 @@ type OpenAPIService struct {
 
 	lastModified time.Time
 
-	specBytes []byte
-	specPb    []byte
-	specPbGz  []byte
+	openapiSpec *spec.Swagger
+	specBytes   []byte
```

Comment:
I'm curious why you have both `...Bytes` here and `...Cache` below. Don't you need only one of the two?